src/pages/UnlockedWallet/OverviewPage/index.tsx

```diff
@@ -71,11 +70,7 @@ const OverviewPage = ({ className }: OverviewPageProps) => {
   )
 }
 
-export default styled(OverviewPage)`
-  background-color: ${({ theme }) => theme.bg.background1};
```

Comment:
The overview page was supposed to have the `background1` bg color, but it's now background2. Let's keep your fix for the margin / padding, but revert the background color?

Reply:
ah, damn, I had removed this before our conversation, I'll bring it back.

Reply:
Fixed